### PR TITLE
add a game (cabbage, goat, traveler, wolf)

### PR DESCRIPTION
**Pull request by macikgozm**
Enjoy the game!

### CODE REVIEW

File: src/main/java/Island.java [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+/**
+ * This program is to simulate cabbage, goat, traveler, and wolf game. 
+ * 
+ * It is a basic implementation of Graph Theory in Discrete Math.
+ * 
+ * @author : Mehmet Acikgoz
+ * @date   : September 18, 2018
+ */
+
+public class Island {
```
Comment (fairwinds0099) [inline src/main/java/Island.java]: pls extract Island class from this class

File: src/main/java/Island.java [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+	 * If not, it keeps on asking the same question 
+	 * @param input is the Scanner object just to get a reply from the user.
+	 * @return user's choice. 
+	 */
+
+	public static char menu(Scanner input) {
+		char choice;
+
+		do {
+			System.out.println("1. move CABBAGE and TRAVELLER");
```
Comment (fairwinds0099) [inline src/main/java/Island.java]: modify the method by writing one sout and using /n

File: src/main/java/Island.java [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+	}
+	
+	public int getElement(int index) {
+		return data[index];
+	}
+	
+	
+	@Override
+	public String toString() {
+		String result ="";
```
Comment (fairwinds0099) [inline src/main/java/Island.java]: consider using String result; instead

File: src/main/java/Island.java [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+		for (int i = 0; i < data.length ; i++) {
```
Comment (fairwinds0099) [inline src/main/java/Island.java]: u already know data.length

Comment (macikgozm) [reply src/main/java/Island.java]: This is just a try. This is more procedural rather than OOP. I will do it in OOP design soon.

**Comment by fairwinds0099** [comment]
Sir please try to focus on less complicated algorithms but more on java
design, database interaction, api design etc.

**Pull request merged**